src/afError.js

```diff
@@ -4,10 +4,9 @@ angular.module('angularFormMessages').directive('afError', function () {
     require: '^afFieldWrap',
     scope: true,
     link: function linkFn($scope, elem, attrs, ctrl) {
-
-      $scope.$on('validation', function (event, modelPath, isValid, message) {
+      $scope.$on('validation', function (event, modelPath, errors) {
         if (modelPath === ctrl.modelPath) {
-          $scope.message = isValid ? undefined : message;
+          $scope.errors = errors;
```

Comment:
Ik denk dat er van errors straks nog wel meer dan alleen het message type zal zijn, dus dan krijg je bijv.:
```
messages: [
  { type: 'error', content: 'x', ... },
  { type: 'warning', content: 'y' }
]
```
En dan de types evt als constante die via een provider te configureren zodat je backend iets flexibeler kan worden.
Maar dat is denk ik meer iets voor later.